frontend/src/scenes/cohorts/cohortLogic.tsx

```diff
@@ -104,12 +104,13 @@ export const cohortLogic = kea<cohortLogicType>({
     listeners: ({ actions, values, key }) => ({
         saveCohort: async ({ cohortParams, filterParams }, breakpoint) => {
             let cohort = { ...values.cohort, ...cohortParams } as CohortType
+            const isNewCohort = typeof cohort.id !== 'number'
```

Comment:
There may be other `"new"` strings here. There already was `newPersonModal`, but I removed that.